packages/react-components/src/components/VideoGallery/TogetherModeStream.tsx

```diff
@@ -84,7 +86,13 @@ export const TogetherModeStream = memo(
       togetherModeStreams?.mainVideoStream?.isAvailable,
       onCreateTogetherModeStreamView
     ]);
-
+    // Re-render the component if mainVideoStream isReceiving changes
+    useEffect(() => {
+      setShowLoadingIndicator(
+        !(togetherModeStreams?.mainVideoStream?.isAvailable && togetherModeStreams?.mainVideoStream?.isReceiving)
```

Comment:
Do we need both of these or just isRecieving? I recall the isAvailable flag is to indicate that there is a remote stream that is there but we might not be rendering it but isRecieving is about whether we are locally recieving the stream